src/docbuilder/rustwide_builder.rs

```diff
@@ -225,7 +225,8 @@ impl RustwideBuilder {
                         .prefix("essential-files")
                         .tempdir()?;
                     copy_dir_all(source, &dest)?;
-                    add_path_into_database(&self.storage, "", &dest)?;
+
+                    add_path_into_database(&self.storage, "/-/rustdoc.static/", &dest)?;
```

Comment:
I think we probably don't want the leading `-` in the file storage. Other than the existing static-root files we have everything partitioned into top-level `build-logs`/`rustdoc`/`sources` directories already so adding another `rustdoc.static` directory will be fine.

Reply:
yeah, agreed. 

the leading `-` is only for the routes, 
we can just use `rustdoc.static` as new top-level

Reply:
Also, this would need to be changed once https://github.com/rust-lang/rust/pull/101702 changes the static file directory from the `OUT_DIR` root to a subfolder. 

( which we can handle separately)